contentful/apps
# feat: [LUMOS-131] - add content type hook

## Purpose



- adds `useGetContentTypes` hook for larger lists returned used for populating content types

Ticket: https://contentful.atlassian.net/browse/LUMOS-131

## Approach



## Testing steps



1. Goto a Vercel App enabled/installed space
2. Go through the configuration flow
3. Once at the content type selection step, make sure the content type drop down is still populated

<img width="852" alt="Screenshot 2024-07-24 at 9 07 08 AM" src="https://github.com/user-attachments/assets/07e179f9-17e0-4b7b-846b-83468ea2db9e">
